### PR TITLE
Update Question Helpful Utility

### PR DESCRIPTION
**Pull request by bee-squared**
- Update className of helpful button / manage state
- Send request to db to update helpfulness of question
- Disable question helpfulness onClick
- Style utility component (basic) - convert to text

### CODE REVIEW

File: src/components/qna-components/utility/qna-utility.component.jsx [hunk 1]
```diff
@@ -15,34 +17,57 @@ class Utility extends React.Component {
...
+    () => {
+      const { questionHelpfulClicked } = this.state;
+      this.setState({ questionHelpfulClicked: !questionHelpfulClicked }, () => {
+        fetch(`http://3.134.102.30/qa/question/${questionId}/helpful`,
+          { method: 'PUT' });
+      });
+    });
   }
 
+  // showAddAnswerModal = (e) => {
```
Comment (danielmjordan) [inline src/components/qna-components/utility/qna-utility.component.jsx:35]: Is there a good reason this is commented out?

Comment (bee-squared) [reply src/components/qna-components/utility/qna-utility.component.jsx:35]: @danielmjordan this is in prep for my next step to build out the addAnswer popup modal, good question!

File: src/components/qna-components/utility/qna-utility.component.jsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ class Utility extends React.Component {
     super(props);
     this.state = {
       questionHelpfulness: 0,
+      questionHelpfulClicked: false,
+      // answerModal: false,
```
Comment (danielmjordan) [inline src/components/qna-components/utility/qna-utility.component.jsx:11]: is there a reason this is commented out?

**Review by danielmjordan — APPROVED** [review]
Everything looks good except there are still a few lines of commented out code...Good work Brian, making swift progress amigo!

**Pull request merged**